ts/apps/polaris-cli/src/app/util/packages-utils.ts

```diff
@@ -0,0 +1,23 @@
+import * as fs from 'fs';
+import axios from 'axios';
+
+const POLARIS_REPO = 'polaris-slo-cloud/polaris';
+const PACKAGE_JSON_FILE = 'package.json';
+const NX_ORG = '@nx';
+
+export async function getNxVersion(version: string): Promise<string> {
+    const response = await axios.get(`https://raw.githubusercontent.com/${POLARIS_REPO}/${version}/ts/package.json`);
+    const packageJson = await response.data;
+    return packageJson.devDependencies.nx;
+}
+
+export async function getLatestReleaseVersion(): Promise<string> {
+    const response = await axios.get(`https://api.github.com/repos/${POLARIS_REPO}/releases/latest`);
```

Comment:
do we need to introduce an extra dependency? We could also use `fetch` depending on the node version we are running (for `fetch` 18+ is required)

Reply:
it's not *really* an extra dependency since polaris already uses axios elsewhere and nx can reuse dependencies between packages, but now that node18 is a requirement it makes sense to just use fetch